catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/BackupCommand.java

```diff
@@ -200,7 +201,7 @@ private String getBackupUrl(String coreName) {
                 && System.getProperty("jetty.port") != null
                 && System.getProperty("hostContext") != null) {
               backupUrl =
-                  System.getProperty("urlScheme", "http")
+                  SystemBaseUrl.getProtocol()
```

Comment:
SystemBaseUrl.getProtocol() returns `https://` or `http://`. As it stands now you will end up with something like `https://://` which is not what you want. You could probably use SystemBaseUrl.getBaseUrl() which will give you the protocol, host and port in a string. Not sure what hostContext is here but the SystemBaseUrl class can probably construct the whole url for you.

Reply:
Ah, dang, I wish we'd caught that back on master...

Reply:
@clockard is this an acceptable fix?

Reply:
That will work. I might have gone with this method instead but it will give you the same result as what you have now. `SystemBaseUrl.constructUrl(System.getProperty("hostContext"), false)`